code/script.js

```diff
@@ -222,104 +225,185 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+
+// Randomly selects a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
+
+
+
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+
+  // Presents the board
+  setTimeout(() => generateBoard(), 500) 
+
+  // Selects a secret person
+  setSecret()
+  console.log(secret)
```

Comment:
great that you are using console.log but remember to remove it when project is done.